external/grpc/README.md

```diff
@@ -110,8 +110,9 @@ Please refer to `grpc_greeter_client` and `grpc_route_client` applications under
 
 An important consideration when running gRPC on TizenRT is the stack size allocated for its threads.
 From our initial verfication, we observe that a minimum thread stack size of 16384 bytes is necessary for the main gRPC thread to run successfully on TizenRT.
-Additionally, this gRPC main thread engine uses `pthread_create` internally for its run-time procedures, for which we recommend setting a stack size of 10240 bytes or higher.
-This can be configured via menuconfig at the location `Kernel Features -> Stack size information -> Default pthread stack size`.
+This can be achieved by either using the `pthread_attr_setstacksize` API or configuring the default pthread stacksize via menuconfig, at the location `Kernel Features -> Stack size information -> Default pthread stack size`.
```

Comment:
This configuration affects all of thread including not gRPC threads. Why don't you use gRPC specific config?

Reply:
The line mentions either approaches as options. The grpc example application uses the pthread API to set the stack size.

Reply:
This is a result of using std::thread in certain C++ applications (for example, the two grpc applications). While we can use `pthread_attr_setstacksize`, and then pass this attr to `pthread_create`, there is no corresponding option for `std::thread`. Hence, we are faced with the option of having to raise the default stack size. I agree this is not memory-efficient, but this is the solution we have at present. Please let me know if you find a better alternative.